Everest/Everest/Models/Act.swift

```diff
@@ -13,22 +13,16 @@ import SwiftyJSON
 class Act: NSObject {
 
   var id: String
-  var category: String
```

Comment:
We should leave the category in the Act.

Reply:
I couldn't find a working way to convert, csv file to json with the previous additional info. That's why I dropped the additional info. However, we should still get all desired functionality with slight work on the app side.

Reply:
There is a way to convert excel file to json format which we can push to the FB db.

Reply:
I already tried with excel format and csv format. I couldn't produce the desired results with the excel file.
If you want to to give it a shot, please feel free. If you succeed, feel free to change the database as you see needed.